scripts/update-snippets.ts

```diff
@@ -1,5 +1,3 @@
-#! /usr/bin/env ts-node
```

Comment:
If you remove this, you probably also modify the mode as you did for update-grammar.ts (755 → 644)?